### PR TITLE
Add script for uploading artifacts

### PR DESCRIPTION
**Pull request by Allda**
The script uploads test artifacts using Pyxis API.

JIRA: ISV-883

### CODE REVIEW

File: operatorcert/entrypoints/upload_artifacts.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    if not os.path.isdir(artifacts_dir):
+        LOGGER.warning(f"{artifacts_dir} is not directory")
+        return artifact_paths
+    files = os.listdir(artifacts_dir)
+    LOGGER.debug(f"Found following artifacts: {files}")
+    for artifact in files:
+        if os.path.isfile(os.path.join(artifacts_dir, artifact)):
+            artifact_paths.append(artifact)
+        else:
+            LOGGER.debug(artifact)
```
Comment (MarcinGinszt) [inline operatorcert/entrypoints/upload_artifacts.py:82]: Could you please make this log more meaningful?

File: operatorcert/entrypoints/upload_artifacts.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+    Args:
+        artifacts_dir (str): Path to artifact directory
+
+    Returns:
+        List[str]: List of artifacts files
+    """
+    artifact_paths = []
+    if not os.path.isdir(artifacts_dir):
+        LOGGER.warning(f"{artifacts_dir} is not directory")
```
Comment (MarcinGinszt) [inline operatorcert/entrypoints/upload_artifacts.py:74]: this looks like logger.error

**Review by MarcinGinszt — APPROVED** [review]
Two nitpicks, LGTM

File: operatorcert/entrypoints/upload_artifacts.py (hunk 1 above)
Comment (Allda) [reply operatorcert/entrypoints/upload_artifacts.py:82]: This was just a debug log for testing purposes. The log is deleted now.

File: operatorcert/entrypoints/upload_artifacts.py (hunk 2 above)
Comment (Allda) [reply operatorcert/entrypoints/upload_artifacts.py:74]: In case a preflight doesn't generate any artifacts we should not log this as an error.

**Pull request merged**